added TestTeamLogin

rodenbe4 opened the pull request:
added the following test cases:
-log in team1
-log in team2, while team1 is logged in
-log out team1
-log out team2; team2 is not currently logged in

rodenbe4 committed:
test kayla

rodenbe4 committed:
added TestTeamLogin

testing the following cases:
-login team1
-login team2, while team1 is logged in
-logout team1
-logout team2 (no current user logged in)

rodenbe4 committed:
Merge branch 'master' into login-user

rodenbe4 committed:
added separate unit test file (Team)

Separated unit tests into its own python file

rodenbe4 committed:
edited Team unit tests, User class

added a new test to Team unit test, made minor alterations to User class

rodenbe4 committed:
edited Team and User classes

added User interface; added the User interface to the Team class

rodenbe4 committed:
Added more Team tests

Refactored Team class, added more tests

Mandabee commented on team.py:
```diff
@@ -1,7 +1,17 @@
+    def login(self,username,password):
+        pass
+    def logout(self):
+        pass
```
add:
def displayMenu(self):
     pass

tekkeon commented on team.py:
```diff
@@ -0,0 +1,17 @@
+class Team(user.User):
+    def __init__(self,n):
+        self.name = n
+        self.username = "team"
```
Do we want these generic values at all or should they just not be anything to start with? Or possibly we could have the init method just take in a name, username, and password...?

tekkeon commented on user.py:
```diff
@@ -0,0 +1,12 @@
+import abc
```
I thought I was writing the user interface but alright haha

tekkeon replied:
Also, can we have a naming convention for interfaces like interface_user.py or user_interface.py just so we can distinguish them from each other?

tekkeon commented on tests/ut_team.py:
```diff
@@ -0,0 +1,35 @@
+        self.team1 = team.Team("Name1")
+        self.team2 = team.Team("Name2")
+    def test_setTeamName(self):
+        self.assertEquals("Team name changed to n1", self.team1.setTeamName("n1"), "Error: changing team name failed")
```
Should be written like this:

def test_setTeamName(self):
    self.team1.set_team_name("n1")
    self.assertEquals(self.team1, "n1", "Error Message")

Same for below tests. Error messages are printed to console, not returned by these functions.

rodenbe4 committed:
implemented Team class

(mostly) implemented the Team class in its entirety

rodenbe4 committed:
added current_user Team class

added ability to change database.current_user as needed for Team.login() and Team.logout()

rodenbe4 committed:
edited User interface

changed user.py to int_user.py

rodenbe4 committed:
modified Team login() and logout()

changed to use getters and setters for the current_user variable in the database
